Examples/MAX32520/DMA/main.c

```diff
@@ -96,24 +96,21 @@ void example1(void)
 
     if (retval != E_NO_ERROR) {
         printf("Failed MXC_DMA_Init().\n");
-
-        while (1) {}
-    }
-
-    flag = 0;
-    MXC_DMA_MemCpy(dstdata, srcdata, MAX_SIZE, memCpyComplete);
-
-    while (flag == 0) {}
-
-    //Validate
-    if (memcmp(srcdata, dstdata, MAX_SIZE) != 0) {
-        printf("Data mismatch.\n");
-
-        while (1) {}
-
         fail += 1;
-    } else {
-        printf("Data verified.\n");
+    }
+    else {
+    	flag = 0;
+		MXC_DMA_MemCpy(dstdata, srcdata, MAX_SIZE, memCpyComplete);
```

Comment:
@Jake-Carter @ozersa , I will change all return values from 0 and -1 to E_NO_ERROR and E_FAIL. I will also run clang formatter for each example modified in this branch. It might be better to merge them after such change.